src/wp-includes/block-supports/layout.php

```diff
@@ -308,6 +308,26 @@ function wp_get_layout_style( $selector, $layout, $has_block_gap_support = false
 	return '';
 }
 
+/**
+ * Gets classname from last tag in a string of HTML.
+ *
+ * @since 6.2.0
+ * @access private
+ *
+ * @param string $html markup to be processed.
+ * @return string String of inner wrapper classnames.
+ */
+function wp_get_classnames_from_last_tag( $html ) {
+	$tags            = new WP_HTML_Tag_Processor( $html );
+	$last_classnames = '';
+
+	while ( $tags->next_tag() ) {
+		$last_classnames = $tags->get_attribute( 'class' );
+	}
+
+	return (string) $last_classnames;
+}
```

Comment:
Suggestion: I only see a single use of this method. Maybe we should avoid introducing a public method and use `WP_HTML_Tag_Processor` directly.

 The functions are always public in PHP, and we can only deprecate them after the release.

Reply:
@Mamaduka Agreed. Can we instead inline this logic in the `wp_render_layout_support_flag()` function please? This means we can avoid introducing this function.

If there is eventually another change that needs something like this, we can always introduce a new function, but right now it seems unnecessary.